static/app/components/group/groupSummary.tsx

```diff
@@ -28,31 +33,67 @@ const makeGroupSummaryQueryKey = (
   {method: 'POST'},
 ];
 
-export function GroupSummary({groupId}: GroupSummaryProps) {
+export function useGroupSummary(groupId: string) {
   const organization = useOrganization();
-  const {data, isLoading, isError} = useApiQuery<GroupSummaryData>(
+
+  return useApiQuery<GroupSummaryData>(
     makeGroupSummaryQueryKey(organization.slug, groupId),
     {
       staleTime: Infinity, // Cache the result indefinitely as it's unlikely to change if it's already computed
     }
   );
+}
+
+function GroupSummaryFeatureBadge() {
+  return (
+    <StyledFeatureBadge
+      type="experimental"
+      title={t(
+        'This feature is experimental and may produce inaccurate results. Please share feedback to help us improve the experience.'
+      )}
+      tooltipProps={{isHoverable: true}}
+    />
+  );
+}
+
+export function GroupSummaryHeader({groupId}: GroupSummaryProps) {
+  const {data, isLoading, isError} = useGroupSummary(groupId);
+  const isStreamlined = useHasStreamlinedUI();
+
+  if (isError || (!isLoading && !data.headline)) {
+    // Don't render the summary headline if there's an error, the error is already shown in the sidebar
+    // If there is no headline we also don't want to render anything
+    return null;
+  }
+
+  const renderContent = () => {
+    if (isLoading) {
+      return <StyledLoadingIndicator size={16} mini />;
```

Comment:
Use `Placeholder` instead for this and try to match the exact height that the headline text would be